### PR TITLE
Generate validation data

### PR DESCRIPTION
**Pull request by vcanaa**
Tool for generating the validation data. The data was already generated generated and saved in https://drive.google.com/drive/folders/1MgrCHc3jqs1LenSslTPpn6vd-lByKsB0

Instructions or regenerating the data in `validation/README.md`

### CODE REVIEW

File: validation/validation_data.py [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+def main():
+   pass
+
+
+@main.command('download-data')
+@click.option('--east', default=EAST, help='East boundary')
+@click.option('--west', default=WEST, help='West boundary')
+@click.option('--north', default=NORTH, help='North boundary')
+@click.option('--south', default=SOUTH, help='South boundary')
+def download_data(east: str, west: str, north: str, south: str):
```
Comment (marict) [inline validation/validation_data.py:81]: specify units

File: validation/validation_data.py [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+    name = re.sub(r'\bShip Canal Bridge\b', 'SHIP CANAL TRL', name)
+    return name.upper()
+
+
+def create_dir(dir):
+    if not os.path.exists(dir):
+        os.makedirs(dir)
+
+
+NODES: dict[int, Node] = {}
```
Comment (marict) [inline validation/validation_data.py]: nit: move to top

File: validation/validation_data.py [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+    'South Lane Street': 'S LANE ST',
+    'West Montlake Place East': 'WEST MONTLAKE PL E',
+    'East Montlake Place East': 'EAST MONTLAKE PL E',
+    'East North Street': 'E NORTH ST',
+    'West Laurelhurst Drive Northeast': 'WEST LAURELHURST DR NE',
+    'East Laurelhurst Drive Northeast': 'EAST LAURELHURST DR NE',
+    'Terrace Street': 'TERRACE ST',
+}
+
+SKIPPED_ROADS = [
```
Comment (marict) [inline validation/validation_data.py]: why are these skipped?

File: validation/validation_data.py [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+        segment_key = edge.get_segment().key
+        if segment_key in SAMPLE_EDGES:
+            # raise Exception(f'Segment already assigned to another edge: {segment_key}')
+            print(f'Segment already assigned to another edge: {segment_key}')
+            continue
+        
+        SAMPLE_EDGES[segment_key] = edge
+
+
+def load_nodes_and_edges_from_osm(file):
```
Comment (marict) [inline validation/validation_data.py]: nit: missing type hints on some of these headers

**Review by marict — CHANGES_REQUESTED** [review]

File: validation/validation_data.py (hunk 3 above)
Comment (vcanaa) [reply validation/validation_data.py]: These roads exist in OSM but not in Seattle road dataset or I was not able to do the mapping. So I had them skipped.

Comment (marict) [reply validation/validation_data.py]: add a comment explaining this

**Review by marict — APPROVED** [review]

**Pull request merged**